Source/Tools/CommonTools/Linux/Utils/ELFSymbolDatabase.cpp

```diff
@@ -55,7 +55,7 @@ bool ELFSymbolDatabase::FindLibraryFile(fextl::string* Result, const char* Libra
   for (auto& Path : LibrarySearchPaths) {
     const fextl::string TmpPath = fextl::fmt::format("{}/{}", Path, Library);
```

Comment:
This would need to be made non-const for the move to actually occur

Reply:
Interesting, TIL. resolved.